core/test/two-steps.test.ts

```diff
@@ -0,0 +1,27 @@
+import { strictEqual } from 'node:assert'
+import { afterEach, beforeEach, test } from 'node:test'
+
+import { secondStep, setBaseTestRoute } from '../index.js'
+import { cleanClientTest, enableClientTest } from './utils.js'
+
+beforeEach(() => {
+  enableClientTest()
+})
+
+afterEach(async () => {
+  await cleanClientTest()
+})
+
+test('show first side', () => {
+  setBaseTestRoute({ params: {}, route: 'add' })
```

Comment:
If `router.subscribe()` doesn’t work, we will have the same `false`.

We could improve test by something like:

```js
  setBaseTestRoute({ params: {}, route: 'add' })
  strictEqual(secondStep.get(), false)

  showSecondStep() 
  strictEqual(secondStep.get(), false)

  setBaseTestRoute({ params: {}, route: 'categories' })
  strictEqual(secondStep.get(), false)
```